rust/src/api/simple.rs

```diff
@@ -370,20 +371,21 @@ pub fn reset_wallet(encoded_wallet: String) -> Result<String> {
     Ok(serde_json::to_string(&wallet).unwrap())
 }
 
-pub async fn sync_blockchain() -> Result<()> {
-    blindbit::logic::sync_blockchain().await
+pub async fn sync_blockchain(network: String) -> Result<()> {
```

Comment:
(Not strictly relevant to this PR) we should rename this to `get_chain_height` or something.